data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/PstSetpoint.java

```diff
@@ -29,30 +33,45 @@
 @JsonSerialize(using = PstSetPointSerializer.class)
 public final class PstSetpoint extends AbstractSetpointElementaryNetworkAction {
 
+    protected RangeDefinition rangeDefinition;
```

Comment:
why not make this private (+getter) ?

Reply:
Done. I did it that way because I saw it done like this in the parent class AbstractSetpointElementaryNetworkAction